utils/build/docker/dotnet/parametric/ddtracer_version.Dockerfile

```diff
@@ -1,11 +1,10 @@
-
 FROM mcr.microsoft.com/dotnet/sdk:8.0
-
-RUN apt-get update && apt-get install dos2unix
 WORKDIR /app
 
+# required by 'install_ddtrace.sh' to call 'strings' command
+RUN apt-get update && apt-get install -y binutils
```

Comment:
[Note to reviewers]
- removed `dos2unix`, not needed anymore
- add `binutils`, required by `install_ddtrace.sh`